docs/user-guide/ze-usage.md

```diff
@@ -25,21 +25,33 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
 
 ## Usage Tips
 
-Make the best use of the extension with the following tips:
+Make the best use of Zowe Explorer with the following tips:
 
-- **Data sets, USS, and jobs persistence settings:** You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
+### Data sets, USS, and jobs persistence settings
+You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
 
-- **Identify syntax errors with a syntax highlighter:** Zowe Explorer supports a syntax hightlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
+### Identify syntax errors with a syntax highlighter
+Zowe Explorer supports a syntax hightlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
 
-- **Edit a profile**: You can edit existing profiles listed in the **Side Bar** by clicking the profile's **Edit** icon (next to the **Search** icon). The feature lets you modify the information inside your profile.
+### Change the detected language of a file or data set
 
-- **Delete a profile**: In Zowe V1, you can permanently delete profiles by right-clicking the profile and selecting the **Delete Profile** option. The feature deletes the profile from your `.zowe` folder. In Zowe V2, right-click the profile, and select **Delete Profile** to open the configuration file and manually delete the profile.
+If the language for a file or data set opened in Zowe Explorer is not detected correctly, configure Visual Studio Code to use a specific language for a particular file extension. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
```

Comment:
`for a particular file extension` - would `for that file's extension` be clearer?

Reply:
Updated the text so the scenario applies more broadly. Please review.